frontends/web/src/routes/new-settings/components/about/app-version-setting.module.css

```diff
@@ -0,0 +1,16 @@
+.version {
+    color: var(--color-secondary);
+    font-size: 14px;
```

Comment:
forgot: isn't there a CSS variable that you can use for fontsize?

Reply:
Not exactly for 14px. We can go for 16 though, although it'd be a bit too big.. just a bit.

Reply:
I checked there is many hard coded 14px font-sizes.

Let's keep as is, one day we should look at font sizes globally and check if it should be rendered with bigger font on large screens.

Reply:
Actually we do have a 14px called `--size-default`, but this gets overriden by this class

```css
.inner p {
        --size-default: 20px;
    }
```

in `view.module.css`. So that's prob why we have a lot of 14px hardcoded, including here.